apps/showcase/src/app/home/home.template.html

```diff
@@ -1,6 +1,41 @@
 <div class="d-flex flex-column gap-2 align-items-center justify-center flex-fill">
-  <h1>Otter</h1>
-  <p>This is a showcase application.</p>
-  <p>You can find examples of Otter features with some explanations.</p>
-  <img width="500" [src]="'otter-a.svg' | dynamicContent" class="otter-logo">
+  <img width="350" [src]="'otter-a.svg' | dynamicContent" class="otter-logo" alt="Otter">
+  <h1 class="h1 display-1 text-center mt-0 mb-5"><b>Build easily a customizable Angular based application</b></h1>
```

Comment:
I feel the header looks way too big with `display-1`

Reply:
<img width="1777" alt="image" src="https://github.com/AmadeusITGroup/otter/assets/2467012/76eb2527-2364-4c79-9bf5-e03f1b5cac8a">
What do you think?